### PR TITLE
Add case studies to financial-services

### LINKED ISSUE #9773: [COPY UPDATE] financial-services - "Financial services"
Please update financial-services - "Financial services" from the [copy doc](https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Update `/financial-services` adding 2 case studies

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/financial-services
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against [copy doc](https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit#) - two case studies added

## Issue / Card

Fixes #9773 

## Screenshots

image

### CODE REVIEW

File: templates/financial-services/index.html [hunk 1]
```diff
@@ -234,11 +234,58 @@ <h2>Reduce IT operations cost</h2>
...
+          }}
+          <p class="p-heading-icon__title p-heading--4" style=" color: #FFF; line-height: 1.5rem;">Case study</p>
+        </div>
+      </div>
+      <p class="p-heading--4">SBI reduced infrastructure costs with Charmed OpenStack</p>
+      <blockquote class="p-pull-quote--small">
+        <p class="p-pull-quote__quote">Canonical's solution was a third of the price of the other proposals we'd received. The solution is also proving to be highly cost-effective, both from CAPEX and OPEX perspectives.</p>
+        <cite class="p-pull-quote__citation">Georgi Georgiev, CIO at SBI BITS</cite>
+      </blockquote>
+      <p>Charmed OpenStack helped SBI Group reduce OpEx costs and optimise CapEx&ndash;learn how in this case study.</p>
```
Comment (anthonydillon) [inline templates/financial-services/index.html:262]:
```suggestion
      <p>Charmed OpenStack helped SBI Group reduce OpEx costs and optimise CapEx &mdash; learn how in this case study.</p>
```

File: templates/financial-services/index.html [hunk 2]
```diff
@@ -234,11 +234,58 @@ <h2>Reduce IT operations cost</h2>
...
+      <div class="p-heading-icon--muted">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
+            alt="",
+            width="32",
+            height="32",
+            hi_def=True,
+            attrs={"class":"p-heading-icon__img p-heading-icon__img--small"},
+            loading="auto|lazy"
```
Comment (anthonydillon) [inline templates/financial-services/index.html:251]: I think you need to choose one?

File: templates/financial-services/index.html [hunk 3]
```diff
@@ -234,11 +234,58 @@ <h2>Reduce IT operations cost</h2>
...
+            ) | safe
+          }}
+          <p class="p-heading-icon__title p-heading--4" style=" color: #FFF; line-height: 1.5rem;">Case study</p>
```
Comment (anthonydillon) [inline templates/financial-services/index.html:254]: Can you file an issue on Vanilla for this pattern if its commonly used please

File: templates/financial-services/index.html [hunk 4]
```diff
@@ -234,11 +234,58 @@ <h2>Reduce IT operations cost</h2>
...
+      <div class="p-heading-icon--muted">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
+            alt="",
+            width="32",
+            height="32",
+            hi_def=True,
+            attrs={"class":"p-heading-icon__img p-heading-icon__img--small"},
+            loading="auto|lazy"
```
Comment (anthonydillon) [inline templates/financial-services/index.html:275]: Lazy, probably

**Review by anthonydillon — APPROVED** [review]
LGTM with a few comments

File: templates/financial-services/index.html (hunk 3 above)
Comment (bethcollins92) [reply templates/financial-services/index.html:254]: Will do

**Pull request merged**